src/lib/shell/commands/Dns.cpp

```diff
@@ -124,7 +124,7 @@ CHIP_ERROR ResolveHandler(int argc, char ** argv)
     peerId.SetCompressedFabricId(strtoull(argv[0], NULL, 10));
     peerId.SetNodeId(strtoull(argv[1], NULL, 10));
 
-    return Dnssd::Resolver::Instance().ResolveNodeId(peerId, Inet::IPAddressType::kAny);
+    return Dnssd::Resolver::Instance().ResolveNodeId(peerId, Inet::IPAddressType::kAny, true);
```

Comment:
In the upstream PR, please add `/* bypass cache */` comment next to the boolean literal. Alternatively, the community likes using enum classes for such flags, so that the code is self-explanatory. For example,
```
enum class CacheUsage
{
  On,
  Off
};
```
and then:
`Dnssd::Resolver::Instance().ResolveNodeId(peerId, Inet::IPAddressType::kAny, Dnssd::CacheUsage::On)`.